ene/series_manager.py

```diff
@@ -0,0 +1,116 @@
+""" This module handles interactions with Show and Episode objects """
+from ene.persistence.data_access import ShowDataAccess
+from ene.entities import ShowList
+from ene.files import FileManager
+
+
+class SeriesManager:
+    """
+    Manages access to series
+    """
+    def __init__(self, config):
+        self._series = ShowList()
+        self._db = None
+        self._config = config
+
+    def init_db(self, data_home):
```

Comment:
Should inject an instance `ShowDataAccess` instead (remove this method)

Reply:
As per discussion this will be resolved later